### ISSUE TITLE
Audio Conversion Issue After iOS 17 Update: Noise and Distortion in CAF Files

### ISSUE THREAD

**Issue by RajaramTech** [dlutton/flutter_tts#422]
## 🐛 Bug Report

Hi, I have utilized the 'synthesize to file' feature to convert specific text into an audio file in the iOS environment. Everything worked flawlessly before the iOS 17 update. However, after updating my phone to iOS 17, I have encountered an issue where the conversion is no longer producing the expected results. Instead, the output contains only noise and distortion.

I have attached one of the converted files as a reference. I am seeking your assistance in finding a workaround or resolving this issue as quickly as possible.

### Expected behavior

Just like before IOS 17, I expected a perfectly converted CAF audio file.

### Reproduction steps

I have implemented in the following way for IOS:

**String newfileName = "tts.caf";
await flutterTts.setSharedInstance(true);
await flutterTts.awaitSynthCompletion(true);
await flutterTts.synthesizeToFile(
            "Hello World", Platform.isAndroid ? "tts.wav" : newfileName);**

### Configuration

**Version:** 3.8.3

**Platform:** 
- [ ] :iphone: iOS 17
[tts.caf.zip](https://github.com/dlutton/flutter_tts/files/12697713/tts.caf.zip)

**Comment by manhhaiphp** [comment]
I have the same problem. Function **synthesizeToFile** is broken on iOS 17.

**Comment by Colton127** [comment]
I don't think this is a problem with the plugin per se, I think it's just broken in iOS 17. If you google "tts synthesize ios 17 problem" you'll find various posts about it occuring independent of the programming language used.

On second note, "enhanced" and "premium" voices added through iOS settings menu appear to function correctly.

**Comment by RajaramTech** [comment]
Thank you. It is working in my iphone 11. I dint get a chance to check other real devices. Hopefully, it will work on other devices too.
